be/src/vec/exec/join/vhash_join_node.cpp

```diff
@@ -825,6 +873,126 @@ Status HashJoinNode::_process_build_block(RuntimeState* state, Block& block, uin
     return st;
 }
 
+void HashJoinNode::_hash_table_convert_to_partitioned(HashTableVariants& new_hash_table_variants,
```

Comment:
just create new_hash_table_variants in _hash_table_init then these code is not needed.